### PR TITLE
Documentation: updates to Getting Started Guide

### PR DESCRIPTION
**Pull request by gvancuts**

### CODE REVIEW

**Comment by dongyaozu** [comment]
@ywan170  can you have a look?

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -281,6 +281,18 @@ partition. Follow these steps:
 #. From here you can login as root using the password you set previously when
    you installed Clear Linux.
 
+#. (**Optional**) The ``software-defined-cockpit`` bundle installs the
+   ``ioc-cbc-tools`` on the system and activates three ``systemd`` services.
+   Those services do not work on off-the-shelf platforms such as NUCs, UP2.
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: Seems wrong to have software installed and services run that aren't supported on the open source defined hardware platforms.

**Comment by dbkinder** [comment]
Are we waiting for approval by @alekdu to merge this change?

File: doc/getting-started/apl-nuc.rst (hunk 1 above)
Comment (gvancuts) [reply doc/getting-started/apl-nuc.rst]: It is wrong but until we get these service files fixed, this is my best recommendation.

**Comment by alekdu** [comment]
> Looks good to me.
> @alekdu Can you please help double confirm?

Looks good to me too.

File: doc/getting-started/apl-nuc.rst (hunk 1 above)
Comment (alekdu) [reply doc/getting-started/apl-nuc.rst]: Agree, blindly running cbc service and waiting for service to die (30 seconds later) is not good solution. But there isn't really good way to detect IOC chip, unless ACPI table could give us some hints.

Comment (gvancuts) [reply doc/getting-started/apl-nuc.rst]: Is there any entry from the ACPI table we could use for that, something like a PlatformID or anything else? (disclaimer: I really don't know much about ACPI tables)

**Pull request merged**